sorc/ncep_post.fd/PARA_RANGE.f

```diff
@@ -46,4 +46,22 @@ SUBROUTINE PARA_RANGE (N1,N2,NPROCS,IRANK,ISTA,IEND)
       if ( iwork2 > irank ) iend = iend + 1
       return
       end
+      SUBROUTINE PARA_RANGE2 (N1,N2,i1,i2,NPROCS,IRANK,ISTA,IEND,isx,iex)
+
+      implicit none
+      integer,intent(in)  ::  n1,n2,nprocs,irank,i1,i2
+      integer,intent(out) ::  ista,iend,isx,iex
+      integer iwork1, iwork2
+
+      iwork1 = ( n2 - n1 + 1 ) / nprocs
+      iwork2 = mod ( n2 - n1 + 1, nprocs )
+      ista   = irank * iwork1 + n1 + min ( irank, iwork2 )
+      iend   = ista + iwork1 - 1
+      if ( iwork2 > irank ) iend = iend + 1
+      isx=i1
+      iex=i2
```

Comment:
@GeorgeVandenberghe-NOAA Can you explain how this two dimension decomposition method is working? I do not see the i dimension decomposition.

Reply:
I guess only the option of the two dimension decomposition is setup.  So the i direction is still from 1 to im.